### PR TITLE
Update to gradle 5.0

### LINKED ISSUE #561: Migrate to Gradle 5

### PR DESCRIPTION
**Pull request by keyboardsurfer**
Requires Android Studio 3.4 canary from now on.

Fixes #561

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```
Comment (ataulm) [inline gradle/wrapper/gradle-wrapper.properties]: bin?

Comment (keyboardsurfer) [reply gradle/wrapper/gradle-wrapper.properties]: With us starting to migrate to kotlin script soon, I'd rather have docs available as well. See #562

**Comment by keyboardsurfer** [comment]
firebase & crashlytics not necessarily, but the other ones are.

**Comment by nickbutcher** [comment]
Can you update commit headline to not AGP 3.4.0-a5 then LGTM

**Pull request merged**